products/template/build_scripts/export.py

```diff
@@ -24,11 +33,40 @@ def export():
         data = PG_CLIENT.get_table(table_name)
         file_path = OUTPUT_DIR / table_name
         for file_type in file_types:
+            logger.info(
+                f"Exporting table\n\t{table_name}\n\tas a {file_type} to\n\t{OUTPUT_DIR}"
+            )
             if file_type == "csv":
-                logger.info(
-                    f"Exporting table\n\t{table_name}\n\tas a {file_type} to\n\t{OUTPUT_DIR}"
-                )
                 data.to_csv(file_path.with_suffix(".csv"), index=False)
+            elif "shapefile" in file_type:
+                shapefile_directory = OUTPUT_DIR
+                geodata = geospatial.convert_to_geodata(
+                    data,
+                    geometry_column="wkb_geometry",
+                    geometry_format=geospatial.GeometryFormat.wkb,
+                    crs=geospatial.GeometryCRS.wgs_84_deg,
```

Comment:
question: do we normally export our package shapefiles in WGS 84?

Reply:
good question. the COLP data on Open Data [here](https://data.cityofnewyork.us/City-Government/City-Owned-and-Leased-Property-COLP-/fn4k-qyk2/data_preview) looks to be WGS 48 (looking at the numbers in the lat, long, and geom columns)

but COLP in our DB looks to be state plane and I don't see any reprojecting in the export code. so maybe Open Data reprojects after we upload?

Reply:
Perhaps when we abstract to `utils`, we want to be able to explicitly specify a desired projection